recipe/build-r-base.sh

```diff
@@ -455,14 +345,12 @@ Mingw_w64_makefiles() {
     # echo "Running make check-all (up to 3 times, there is some flakiness in p-r-random-tests.R), this will take some time ..."
     # make check-all -j1 > make-check.log 2>&1 || make check-all -j1 > make-check.2.log 2>&1 || make check-all -j1 > make-check.3.log 2>&1
     cd installer
-    make imagedir
+    R_PKGS_RECOMMENDED= make imagedir
```

Comment:
We'd probably need this as
```suggestion
    make R_PKGS_RECOMMENDED= imagedir
```
too, right? Because I think those come from https://github.com/wch/r-source/blob/tags/R-4-3-3/share/make/vars.mk#L12 (using `=` not `?=`).

Reply:
If need be, we could also set
```sh
MAKE="${MAKE-make} R_PKGS_RECOMMENDED"
```
and use `${MAKE}` instead everywhere.

Reply:
Or `MAKE_ARGS`.